ait/core/cmd.py

```diff
@@ -557,6 +558,8 @@ def YAMLCtor_include(loader, node):  # noqa
     # Get the path out of the yaml file
     name = os.path.join(os.path.dirname(loader.name), node.value)
     data = None
+    # DEBUG
+    log.debug(f'NAME of File: {name}')
```

Comment:
Could this either be removed, or make the debug statement more meaningful?  Seeing it in a log as is wouldn't be too helpful.